mmv1/templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -1,6 +1,13 @@
 func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+	oo, nn := d.GetChange("rrdatas")
```

Comment:
We only need to get the change on the whole list if we are comparing AAAA types I believe.

We can move this into the `if d.Get("type") == "AAAA" {` block

Reply:
Will update accordingly